packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -78,13 +76,45 @@ const SupportListItem = ({ item }: { item: SupportItem }) => {
   );
 };
 
-export const SupportButton = ({ children }: PropsWithChildren<Props>) => {
-  const { items } = useSupportConfig();
+const createDisplayedItems = (
+  showCondition: Boolean | undefined,
+  generalItems: SupportItem[],
+  supportItems: SupportItem[] | undefined,
+) => {
+  if (showCondition && supportItems) {
+    return supportItems;
+  }
+  if (supportItems) {
+    return [...supportItems, ...generalItems];
+  }
+  return generalItems;
+};
+
+type Props = {};
 
+type SupportButtonProps = {
+  title?: string;
+  supportItems?: SupportItem[];
+  hideDefaultItems?: Boolean;
```

Comment:
Btw., You should prefer the lower case `boolean` type. I wonder weather we could have a lint for that (including `String`).